demo/packages/tabs/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,229 @@
+
+### Lessons learned
+
+ING started using web components very early on, and Lion is not the first component library we built. So you may wonder, how are these components different from the previous generation?
+
+Lion is built from the ground up allow for accessibility and extendability as we learned that these things are almost impossible to change/achieve at a later point. We'd like to highlight a few of the lessons learned while making the new set of components.
+
+- Not everything has to be a Web Component, for certain functionality regular JavaScript is better.
+- Stay as close to native semantic HTML elements as possible.
+- Incorporate accessibility from the start.
+- Not everything needs to be in the shadow DOM, this is especially important for aria relations and accessibility.
+- UI components are hard
+
+> Everyone should use web components but not everyone should write them
+
+*Erik*
+
+## 🙋‍ Looking For Contributors/Adopters!
+
+In order for the open sourcing of Lion to be a success, we need your helping hand! What can you do?
+
+### Adopt Lion!
+
+Mainly you can help by starting using Lion! 👍
+
+The more extending components and users there are, the more stable the Lion base will become, which helps everyone who uses Lion.
+
+Consider:
+- Promoting Lion within your company
+- Showing Lion to your Tech Architects
+- Bring Lion up if there is a plan to start (codifying) your Design System
+
+- Extending Lion to make a web component version of your favorite design systems, such as: [Bulma](https://bulma.io/expo/), [Bootstrap](https://getbootstrap.com/), [Material](https://material.io/design/), [Bolt](https://boltdesignsystem.com/), [Argon](https://demos.creative-tim.com/vue-argon-design-system/documentation/), [Tailwind](https://tailwindcss.com/)
+- Or even suggest Lion to them!
+
+### Contribute to Lion!
+
+If you want to do more you can help Lion directly by:
+
+- Reporting/fixing issues
+- Working on a completely new component (start with an issue for discussion first)
+- Improving documentation
+- ... every contribution counts! Even typo fixes in the docs 🤓
+
+
+## 🔨 Extend the components
+
+You can use Lion as the base to implement your own codified design system.
+
+Let see how Alfred's story would have panned out if he would have chosen Lion instead.
+
+First install what you need
+
+```bash
+npm i lit-element @lion/tabs
+```
+
+Create `alfred-tabs` by reusing the functionality of Lion. This gives Alfred all the functionality and accessible core that he needs for his custom tabs component.
+
+```js
+import { css } from 'lit-element';
+import { LionTabs } from '@lion/tabs';
+
+export class AlfredTabs extends LionTabs {
+  static get styles() {
+    return [
+      super.styles,
+      css`
+        /* my stylings */
+      `
+    ];
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this._setupFeature();
+  }
+
+  _setupFeature() {
+    // being awesome
+  }
+}
```

Comment:
Perhaps it might be a good idea to extend these JavaScript snippets to also include the `customElements.define(...)` call.

I think, for many people viewing this post, this will be the first time they encounter web components, so it might a good idea to add the CustomElementRegistry call incase they decide to literally copy these snippets.